components/picker/stories/template.js

```diff
@@ -12,17 +12,24 @@ import { when } from "lit/directives/when.js";
 
 import "../index.css";
 
+/**
+ * Template for Picker only (no popover or help text).
```

Comment:
Your extra comments are great- definitely clears up the `Template` and `Picker` exports. 👍